components/automate-ui/src/app/pages/+compliance/+scanner/containers/nodes-list/nodes-list.component.ts

```diff
@@ -29,6 +29,8 @@ export class NodesListComponent implements OnInit, OnDestroy {
 
   showNodeResults = false;
 
+  failedNode: object;
```

Comment:
New failedNode variable to store all the information for the current Node Error requested to be viewed.